build-constraints.yaml

```diff
@@ -6738,96 +6738,1247 @@ packages:
+        - BiobaseFasta < 0 # tried BiobaseFasta-0.4.0.1, but its *library* requires the disabled package: streaming
+        - BiobaseFasta < 0 # tried BiobaseFasta-0.4.0.1, but its *library* requires the disabled package: streaming-bytestring
+        - BiobaseTypes < 0 # tried BiobaseTypes-0.2.1.0, but its *library* requires the disabled package: streaming
+        - BiobaseXNA < 0 # tried BiobaseXNA-0.11.1.1, but its *library* requires the disabled package: BiobaseTypes
+        - BiobaseXNA < 0 # tried BiobaseXNA-0.11.1.1, but its *library* requires the disabled package: DPutils
+        - BiobaseXNA < 0 # tried BiobaseXNA-0.11.1.1, but its *library* requires the disabled package: PrimitiveArray
+        - ChannelT < 0 # tried ChannelT-0.0.0.7, but its *library* does not support: mmorph-1.2.0
+        - DAV < 0 # tried DAV-1.3.4, but its *library* requires the disabled package: http-client-tls
+        - DAV < 0 # tried DAV-1.3.4, but its *library* requires the disabled package: xml-hamlet
+        - DPutils < 0 # tried DPutils-0.1.1.0, but its *library* requires the disabled package: streaming
+        - DPutils < 0 # tried DPutils-0.1.1.0, but its *library* requires the disabled package: streaming-bytestring
+        - ENIG < 0 # tried ENIG-0.0.1.0, but its *library* requires the disabled package: unicode-transforms
+        - Frames < 0 # tried Frames-0.7.2, but its *library* does not support: base-4.16.0.0
+        - Frames < 0 # tried Frames-0.7.2, but its *library* does not support: ghc-prim-0.8.0
+        - GLUT < 0 # tried GLUT-2.7.0.16, but its *library* requires the disabled package: OpenGL
+        - HDBC < 0 # tried HDBC-2.4.0.3, but its *library* does not support: time-1.11.1.1
+        - HDBC-session < 0 # tried HDBC-session-0.1.2.0, but its *library* requires the disabled package: HDBC
+        - HMock < 0 # tried HMock-0.5.0.0, but its *library* does not support: base-4.16.0.0
+        - HMock < 0 # tried HMock-0.5.0.0, but its *library* does not support: template-haskell-2.18.0.0
+        - HStringTemplate < 0 # tried HStringTemplate-0.8.8, but its *library* does not support: bytestring-0.11.1.0
+        - HStringTemplate < 0 # tried HStringTemplate-0.8.8, but its *library* does not support: semigroups-0.20
+        - HStringTemplate < 0 # tried HStringTemplate-0.8.8, but its *library* does not support: template-haskell-2.18.0.0
+        - HStringTemplate < 0 # tried HStringTemplate-0.8.8, but its *library* does not support: time-1.11.1.1
+        - HaTeX < 0 # tried HaTeX-3.22.3.0, but its *library* does not support: bytestring-0.11.1.0
+        - HaTeX < 0 # tried HaTeX-3.22.3.0, but its *library* does not support: hashable-1.4.0.0
+        - HaXml < 0 # tried HaXml-1.25.7, but its *library* does not support: base-4.16.0.0
+        - HaXml < 0 # tried HaXml-1.25.7, but its *library* does not support: bytestring-0.11.1.0
```

Comment:
- https://github.com/HaXml/HaXml/issues/4

Reply:
HaXml fixed in 1.25.8